### PR TITLE
Add vcr integration tests

### PR DESCRIPTION
**Pull request by ashulmanWeWork**

### CODE REVIEW

**Comment by sshah-wework** [comment]
I think you meant to add as as reviewers and not assignees 😛

File: test/test_job.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+@fixture(scope='class')
+def set_valid_env():
+    os.environ[MarquezClient.MARQUEZ_HOST_KEY] = "localhost"
+    os.environ[MarquezClient.MARQUEZ_PORT_KEY] = "8080"
+
+
+@fixture(scope='class')
+def marquez_client(set_valid_env):
```
Comment (sshah-wework) [inline test/test_job.py:17]: hm, do you want a `class` scoped client? it is being mutated across tests

Comment (ashulmanWeWork) [reply test/test_job.py:17]: Good catch. Will adjust to method scope.

File: test/test_job.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    return MarquezClient()
+
+
+@fixture(scope='class')
+@vcr.use_cassette('fixtures/vcr/test_job/namespace_for_job.yaml')
+def namespace(marquez_client):
```
Comment (sshah-wework) [inline test/test_job.py:23]: I don't see this method called elsewhere

Comment (ashulmanWeWork) [reply test/test_job.py:23]: It's a test fixture - used in `test_create_job(..)`

**Review by roaraya8 — APPROVED** [review]
LGTM

**Pull request merged**